arabic/salamoAlikom.md

```diff
@@ -9,3 +9,5 @@
 [اريد ان اتعلم المزيد](learn/learn.md)
 
 [اريد ان احلم](dream/dream.md)
+
+(passion/passion.md) [اريد ان اجد شغفي في الحياه]
```

Comment:
By the way, I'm not sure if this happens for you, but my text editor displays Arabic text backwards compared to how it is displayed on GitHub, so perhaps that is the source of the problem? Regardless, if you format this link like the links above it, it should work.